Doc/library/fileinput.rst

```diff
@@ -54,7 +54,7 @@ provided by this module.
 The following function is the primary interface of this module:
 
 
-.. function:: input(files=None, inplace=False, backup='', bufsize=0, mode='r', openhook=None)
+.. function:: input(files=None, inplace=False, backup='', mode='r', openhook=None)
```

Comment:
All parameters after the removed one should be keyword-only.

Reply:
Done.